client/integration-tests/security/pom.xml

```diff
@@ -91,5 +95,30 @@
         <quarkus.package.type>native</quarkus.package.type>
       </properties>
     </profile>
+    <profile>
+      <id>resteasy-reactive</id>
```

Comment:
Maybe just `rest` given that the artifact is `quarkus-rest` (we dropped the reactive some versions ago)

Reply:
That's just the profile name that we use for tests.
It would be nice to rename them though. But maybe it would be better to do it in a specific PR.